### ISSUE TITLE
Climate data on ClimMob reports 

### ISSUE THREAD

**Issue by kauedesousa** [AgrDataSci/ClimMob-analysis#105]
Currently the report uses the climate data for the whole "season" meaning the first registration date to the last submission date in the last data collection moment. This is a good approach to see how was the climate during the whole experiment, but not to be used in the PL trees. 

To avoid misleading results, I suggest to capture the climate for the first 60 days after planting. This would be done by looking for the variable planting_date in the dataset or in absence of it, by the registration date (which could be misleading as well). 

I work in this approach or any other suggestion in the coming days.

**Comment by jacobvanetten** [comment]
Eventually, the number of days should depend on the crop and eventually the
real phenology...

**Comment by kauedesousa** [comment]
Yes. This is something that we can work with @BrandonMrBot in the UI. But to avoid a lot of click options in the report request, we could implement something that I was discussing with @RJManners. Which is making a small dataset with important crop phenological thresholds reported in the literature or by interviewing breeders. Then the process is less arbitrary, supported by literature or expert feedback, and also tailored for each crop (or even mega-environments).

**Comment by kauedesousa** [comment]
Phenological thresholds could be something for a MSc student or a short consultancy.

**Comment by jacobvanetten** [comment]
You could calculate crop-specific phenological thresholds by taking the
geometric mean of the EcoCrop phenological variables, like we have done in
our article "Are agricultural researchers working on the right crops to
enable food and nutrition security under future climates?"
